indexs.js

```diff
@@ -1,110 +1,50 @@
+const express = require('express')
+const mongoose = require('mongoose')
+const User = require('./user');
+var url = "mongodb+srv://prateek:java1234@cluster0.vkgzh.mongodb.net/nodedb?retryWrites=true&w=majority"
```

Comment:
- try to use let instead of the var
- implement env file (environment file) to avoid hardcoded values